src/System.Net.Security/tests/Scripts/kdc.conf.ubuntu

```diff
@@ -0,0 +1,16 @@
+[kdcdefaults]
+    kdc_ports = 750,88
+
+[realms]
+    TEST.COREFX.NET = {
+        database_name = /var/lib/krb5kdc/principal
+        admin_keytab = FILE:/etc/krb5kdc/kadm5.keytab
+        acl_file = /etc/krb5kdc/kadm5.acl
+        key_stash_file = /etc/krb5kdc/stash
+        kdc_ports = 750,88
```

Comment:
This is default kdc.conf file generated with the KDC installation on Ubuntu/Debian.